Assets/Modules/0_Global/Scripts/Utils.cs

```diff
@@ -128,5 +129,24 @@ public static float Clamp(this float value, float min, float max)
             }
             return value;
         }
+
```

Comment:
Commentaires sur la fonction
```suggestion
/// <summary>
        /// TODO 
        /// <example> Example(s):
        /// <code>
        ///     TODO
        /// </code>
        /// </example>
        /// </summary>
        /// <param name="imediate"> TODO</param>
        /// <returns>
        /// TODO
        /// </returns>
```